src/platforms/android/common/performance/instrumentation/apollo.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Apollo Integration
+sidebar_order: 20
+description: "Learn how to capture the performance of Apollo GraphQL client."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+Sentry Apollo integration provides the `SentryApolloInterceptor`, which creates a span for each outgoing HTTP request executed with an [Apollo Android](https://github.com/apollographql/apollo-android) GraphQL client.
+
+### Install
```

Comment:
Skipping heading levels breaks the right hand sidebar. Always start at H2 in the body of your content.
```suggestion
## Install
```